dll/kernel32.cpp

```diff
@@ -800,9 +977,12 @@ namespace kernel32 {
 	void *WIN_FUNC GetProcAddress(void *hModule, char *lpProcName) {
 		DEBUG_LOG("GetProcAddress: %s from %p\n", lpProcName, hModule);
 
-		if ((unsigned int)hModule == 1) {
-			if (strcmp(lpProcName, "IsProcessorFeaturePresent") == 0) return (void *) IsProcessorFeaturePresent;
-		}
+		if (strcmp(lpProcName, "IsProcessorFeaturePresent") == 0) return (void *) IsProcessorFeaturePresent;
+		// if (strcmp(lpProcName, "InitializeCriticalSectionEx") == 0) return (void *) InitializeCriticalSectionEx;
```

Comment:
why commented out? would be nice if these could make use of `resolveStubByName`, maybe passing `dllName == nullptr` and special-casing that to circumvent the strcasecmp checks until we add proper handling of module handles.

Reply:
commented out because it segfaulted
```
LoadLibraryExW: api-ms-win-core-synch-l1-2-0
GetProcAddress: InitializeCriticalSectionEx from 0x100005
Segmentation fault (core dumped)
```